pkgs/development/r-modules/default.nix

```diff
@@ -546,6 +546,8 @@ let
     bayesWatch = [ pkgs.boost.dev ];
     clustermq = [  pkgs.pkg-config ];
     coga = [ pkgs.gsl.dev ];
+    deepSNV = with pkgs; [ xz.dev bzip2.dev zlib.dev ];
```

Comment:
Since the PR fixing deepSVN has been merged, you could remove this commit from this PR, no?